docs/data/material/varnish/varnish-theme/ThemeValuesDemo.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { getTheme } from '@allenai/varnish2';
+import List from '@mui/material/List';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemText from '@mui/material/ListItemText';
+import Collapse from '@mui/material/Collapse';
+import ExpandLess from '@mui/icons-material/ExpandLess';
+import ExpandMore from '@mui/icons-material/ExpandMore';
+
+export default function Demo() {
+  const theme = getTheme();
+  const topLevelKeys = Object.keys(theme);
+
+  const [open, setOpen] = React.useState<string[]>([]);
+
+  const handleClick = (key: string) => {
+    if (open.includes(key)) {
+      const idx = open.indexOf(key);
+      open.splice(idx, 1);
+      setOpen([...open]);
+    } else {
+      setOpen([...open, key]);
+    }
+  };
+
+  return (
```

Comment:
why not use the same code from here: https://github.com/allenai/varnish/blob/main/apps/varnish-demo/src/pages/Theme/ThemeValues.tsx

Reply:
the way that the AntD Collapse component works is different than how the recommended ListItem + Collapse in Material UI works, I followed the best practice of the Material List Item collapsable button here - https://mui.com/material-ui/react-list/#nested-list 

I did just find this Accordian component however - https://mui.com/material-ui/react-accordion/ 
Will see if it makes sense to retrofit with that component.

Reply:
Oh that makes sense. I think it could look better.... But this is way lower pri than the other tickets... So I recommend circling back... Unless you prefer to do it while it's fresh.

Reply:
Accordion does clean up the code pretty nicely, made the switch :)

Reply:
can you update the screen shot in this pr?